src/main/java/lotto/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+    private static final int LOTTO_MIN_NUMBER = 1;
+    private static final int LOTTO_MAX_NUMBER = 45;
+    private static final int LOTTO_MAX_SIZE = 6;
+    private static List<LottoNumber> lottoList;
+
+    static {
+       lottoList = IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
+           .boxed()
+           .map(LottoNumber::new)
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요! 😉
```suggestion
           .mapToObj(LottoNumber::new)(LottoNumber::new)
```

Reply:
반영하였습니다 👍

Reply:
반영하였습니다 👍